Cargo.toml

```diff
@@ -26,7 +26,7 @@ tokio = { version = "1", features = ["full"] }
 toml = "0.8.19"
 
 [target.'cfg(target_os = "linux")'.dependencies]
-mia-installer = { git = "https://github.com/gevulotnetwork/mia.git", tag = "mia-installer-0.2.3" }
```

Comment:
I've created a new tag, you can use `tag = mia-installer-0.2.4`

Reply:
done